libs/anglify/src/modules/text-area/text-area.component.ts

```diff
@@ -0,0 +1,66 @@
+import { AfterViewInit, Component, ContentChild, ContentChildren, Inject, Input, QueryList, Self, ViewChild } from '@angular/core';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { DEFAULT_TEXT_AREA_SETTINGS, TEXT_AREA_SETTINGS } from './text-area-settings.token';
+import { EntireTextAreaSettings } from './text-area.interface';
+import { createSettingsProvider } from '../../factories/settings.factory';
+import { BooleanLike } from '../../utils/interfaces';
+import { SlotDirective } from '../common/directives/slot/slot.directive';
+import { InputComponent } from '../input/input.component';
+import { InputDirective } from '../input/input.directive';
+import { InputAppearance } from '../input/input.interface';
+
+@UntilDestroy()
+@Component({
+  selector: 'anglify-text-area',
+  templateUrl: './text-area.component.html',
+  styleUrls: ['./text-area.component.scss'],
+  providers: [createSettingsProvider<EntireTextAreaSettings>('anglifyTextAreaSettings', DEFAULT_TEXT_AREA_SETTINGS, TEXT_AREA_SETTINGS)],
+})
+export class TextAreaComponent implements AfterViewInit {
+  @ContentChildren(SlotDirective) public readonly slots?: QueryList<SlotDirective>;
+  @ContentChild(InputDirective) public readonly input?: InputDirective;
+  @ViewChild('anglifyInput', { read: InputComponent }) public anglifyInput!: InputComponent;
+
+  @Input() public label?: string;
+  @Input() public appearance: InputAppearance = this.settings.appearance;
+  @Input() public hint?: string;
+  @Input() public persistentHint: BooleanLike = this.settings.persistentHint;
+  @Input() public alwaysFloatingLabel: BooleanLike = this.settings.alwaysFloatingLabel;
+  @Input() public hideDetails: BooleanLike = this.settings.hideDetails;
+  @Input() public counter: BooleanLike = this.settings.counter;
+  @Input() public error?: string;
+
+  public constructor(@Self() @Inject('anglifyTextAreaSettings') public settings: EntireTextAreaSettings) {}
+
+  public ngAfterViewInit() {
+    this.anglifyInput.onInputClick.pipe(untilDestroyed(this)).subscribe(() => this.input?.elementRef.nativeElement.focus());
+
+    if (this.input) {
+      this.input.disabled$.pipe(untilDestroyed(this)).subscribe(disabled =>
+        setTimeout(() => {
+          this.anglifyInput.disabled = disabled;
+        }, 0)
```

Comment:
I saw this somewhere else already, maybe make sure to note for the other places too, if you want to skip to the next tick, you probably need to put `1` instead of `0` here for better safety.

Reply:
The setTimeout bothers me everywhere. As soon as we know a better way, we should refactor everything. But at the moment I can't think of anything